src/components/select/select-list/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './select-list.component';
```

Comment:
If this isn't a component our consumers should be using, we shouldn't have an export for it.

e.g. we don't want to support

- `import SelectList from 'carbon-react/lib/components/select/select-list`

When using it internally we should use the `select-list.component` directly.